manifests/init.pp

```diff
@@ -331,6 +331,12 @@
   Integer[0] $cluster_migration_barrier                          = 1,
   Hash[String[1], Hash] $instances                               = {},
 ) inherits redis::params {
+  $masterauth_unsensitive = if $masterauth =~ Sensitive {
+    $masterauth.unwrap
+  } else {
+    $masterauth
+  }
```

Comment:
Where is this used?

Reply:
Good Question.  It's used only in `manifests/instance.pp`, but not in `manifests/init.pp`.  I will remove these Lines.